src/nvim/screen.c

```diff
@@ -340,8 +340,12 @@ void update_screen(int type)
   if (need_highlight_changed)
     highlight_changed();
 
-  if (type == CLEAR) {          /* first clear screen */
-    screenclear();              /* will reset clear_cmdline */
+  if (type == CLEAR) {          // first clear screen
+    screenclear();              // will reset clear_cmdline
+    if (curbuf->terminal) {
+      terminal_resize(curbuf->terminal, curwin->w_width - curwin_col_off(),
+                      curwin->w_height, true);
```

Comment:
Ah, so it only resizes a _focused_ terminal. Then the plural language isn't needed in the help doc, right?